react/src/components/ProjectResourcePolicyList.tsx

```diff
@@ -115,31 +117,44 @@ const ProjectResourcePolicyList: React.FC<
       fixed: 'left',
       sorter: (a, b) => localeCompare(a?.name, b?.name),
     },
-    supportMaxVfolderCount && {
-      title: t('resourcePolicy.MaxVFolderCount'),
-      dataIndex: 'max_vfolder_count',
-      key: 'max_vfolder_count',
-      render: (text: ProjectResourcePolicies) =>
-        _.toNumber(text) === 0 ? '∞' : text,
-      sorter: (a, b) =>
-        numberSorterWithInfinityValue(
-          a?.max_vfolder_count,
-          b?.max_vfolder_count,
-          0,
-        ),
-    },
-    supportMaxQuotaScopeSize && {
-      title: t('resourcePolicy.MaxQuotaScopeSize'),
-      dataIndex: 'max_quota_scope_size',
-      key: 'max_quota_scope_size',
-      render: (text) => (text === -1 ? '∞' : bytesToGB(text)),
-      sorter: (a, b) =>
-        numberSorterWithInfinityValue(
-          a?.max_quota_scope_size,
-          b?.max_quota_scope_size,
-          -1,
-        ),
-    },
+    supportMaxVfolderCount
+      ? {
+          title: t('resourcePolicy.MaxVFolderCount'),
+          dataIndex: 'max_vfolder_count',
+          key: 'max_vfolder_count',
+          render: (text: ProjectResourcePolicies) =>
+            _.toNumber(text) === 0 ? '∞' : text,
+          sorter: (a, b) =>
+            numberSorterWithInfinityValue(
+              a?.max_vfolder_count,
+              b?.max_vfolder_count,
+              0,
+            ),
+        }
+      : {},
+    supportMaxQuotaScopeSize
+      ? {
+          title: t('resourcePolicy.MaxQuotaScopeSize'),
+          dataIndex: 'max_quota_scope_size',
+          key: 'max_quota_scope_size',
+          render: (text) => (text === -1 ? '∞' : bytesToGB(text)),
+          sorter: (a, b) =>
+            numberSorterWithInfinityValue(
+              a?.max_quota_scope_size,
+              b?.max_quota_scope_size,
+              -1,
+            ),
+        }
+      : {},
+    supportMaxNetworkCount
+      ? {
+          title: t('resourcePolicy.MaxNetworkCount'),
+          dataIndex: 'max_network_count',
+          key: 'max_network_count',
+          render: (text) => (text === -1 ? '∞' : text),
```

Comment:
I updated one line to display infinity symbol when the value is set to -1, which is regarded as unlimited, for better user experience.